src/molara/Structure/io/importer.py

```diff
@@ -108,30 +108,19 @@ def load(self) -> Molecules:
         with open(self.path, encoding="utf-8") as file:
             lines = file.readlines()
 
-        molecules.add_molecule(
-            self._molecule_from_xyz(lines),
-        )
         num_atoms = int(lines[0])
-
-        # Read in for a single xyz file
-        # Goes on if file has more than one structure stored
-        if len(lines) <= 2 + num_atoms:
-            return molecules
-        first_line_after_first_mol = lines[2 + num_atoms].strip()
-        if not first_line_after_first_mol.isdigit():
-            return molecules
-
         finished = False
-        max_mols = 10000
+        max_mols = int(1e100)
```

Comment:
Isn't that a bit extreme?

Reply:
We could do the same maximum as LAMMPS, i.e., 2e63 😃